src/unix/fs.c

```diff
@@ -49,6 +49,7 @@
     defined(__FreeBSD__)    ||                                            \
     defined(__OpenBSD__)    ||                                            \
     defined(__NetBSD__)
+# include <sys/syslimits.h> /* PATH_MAX, IOV_MAX */
```

Comment:
@TheAlphaNerd You run another CI against just including `limits.h` in this file (without `sys/syslimits.h`), unconditionally, and across platforms, right? I have to admit I can’t really believe FreeBSD ignores POSIX on something like where `PATH_MAX` is defined…